src/mam4xx/ndrop.hpp

```diff
@@ -47,5 +47,51 @@ void get_aer_num(const Diagnostics &diags, const Prognostics &progs,
   naerosol[mode_idx] = max(naerosol[mode_idx], vaerosol * num2vol_ratio_max);
   naerosol[mode_idx] = min(naerosol[mode_idx], vaerosol * num2vol_ratio_min);
 }
+
+KOKKOS_INLINE_FUNCTION
+void explmix(
+    int pver,     // number of levels
+    Real q[4],    // number / mass mixing ratio to be updated [# or kg / kg]
```

Comment:
I know that these should not be the number 4, but placeholder to check the tests while we discuss the solution.

Reply:
How are these arrays sized in the Fortran code?

Reply:
They are sized based on `pver`: 
```
real(r8), intent(out) :: q(pver)
```

Reply:
Okay, that makes sense. But unfortunately, I don't think this will work as is, since `q` is an array of size 4, and our typical `nlev` is 72. I wonder if we need to use `ColumnView`s instead of arrays for this function.

Reply:
We might have to take a look at the function/subroutine that calls this one to figure out how best to pass the data in.